v/vault/build_info.json

```diff
@@ -2,14 +2,22 @@
   "maintainer": "vinodk99",
   "package_name": "vault",
   "github_url": "https://github.com/hashicorp/vault.git",
-  "version": "v1.13.1",
+  "version": "v1.16.2",
   "default_branch": "main",
   "package_dir": "v/vault/",
   "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
   "docker_build": true,
   "use_non_root_user": "false",
   "validate_build_script": true,
-  "build_script": "vault_ubi_8.7.sh",
+  "build_script": "vault_ubi_9.3.sh",
+  "v*.*.*": {
```

Comment:
shouldn't' this be at the end of file else it will satisfy all patterns.

Reply:
Also update specific section for ""v1.13.*,v1.14.*":" with new dockerfile and script